src/mono/wasm/runtime/cancelable-promise.ts

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 import { _lookup_js_owned_object } from "./gc-handles";
-import { createPromiseController, loaderHelpers } from "./globals";
+import { createPromiseController, loaderHelpers, mono_assert } from "./globals";
```

Comment:
the diff viewer shows a mono_assert import being added to this and some other files without being used, won't that fail eslint? is the diff missing the uses?

Reply:
That is correct, the explanation is that before it was kind of virtual global macro as part of rollup. Now it's a real function.

Reply:
See src\mono\wasm\runtime\types\consts.d.ts and src\mono\wasm\runtime\rollup.config.js